.release/ci.hcl

```diff
@@ -0,0 +1,277 @@
+schema = "1"
+
+project "consul-template" {
+  team = "consul-template"
```

Comment:
Hey @claire-labry, while you're here looking..

Do you know which `team` this references and does it need to match up with it?

Reply:
Seems it currently isn't used. Adding the comment from the PR to remind myself.

Reply:
@eikenb I totally missed this! So for Vault-Eco products they use `vault` for (team)[https://github.com/hashicorp/vault-ssh-helper/blob/1dba530b35a1ed036227d286d0d266006a2cde7f/.release/ci.hcl#L4] So in this case, it would make sense to put under `consul`